src/test/java/outcomes/process_executor/go/no_find_executable_file_by_directory/NoFindExecutableFileByDirectory.java

```diff
@@ -0,0 +1,27 @@
+package outcomes.process_executor.go.no_find_executable_file_by_directory;
+
+import org.hyperskill.hstest.common.FileUtils;
+import org.hyperskill.hstest.dynamic.input.DynamicTestingMethod;
+import org.hyperskill.hstest.testcase.CheckResult;
+import org.hyperskill.hstest.testing.TestedProgram;
+import outcomes.base.ContainsMessage;
+import outcomes.base.UserErrorTest;
+
+
+public class NoFindExecutableFileByDirectory extends UserErrorTest {
+
+    @ContainsMessage
+    String m =
+            "Error in test #1\n" +
+                    "\n" +
+                    "Cannot find a file to execute your code in directory \"" + FileUtils.cwd() +
+                    "/src/test/java/outcomes/process_executor/go/no_find_executable_file_by_directory\".";
+
+    @DynamicTestingMethod
+    CheckResult test() {
+        TestedProgram main = new TestedProgram(
+                FileUtils.cwd() + "/non_existent_directory");
```

Comment:
It's not needed to prepend `TestedProgram`, argument with `FileUtils.cwd()` - it will assume the same. So you can write just `new TestedProgram("non_existent_directory")`. And, I assumed that you need to write `new TestedProgram("dir1.dir2")` to reference `cwd/dir1/dir2`, using a dot.

It's done like that because of one little thing: the same tests should be able to test Java and Go/JS/Bash as well. And we need the same referencing style. So, when testing Java `new TestedProgram("dir1.dir2")` would mean "search in package dir1.dir2" and when testing external process it would mean "search in directory cwd/dir1/dir2". So, please, get rid of slashes and "cwd" in `TestedProgram`'s arguments. Slashes are not supported because such thing will make tests non-language independent (for example, such tests will not be able to test Java program, but should be able to do so).

Also, please add more tests with `ProcessExecutor`. I see that there were no tests with `ProcessExecutor` in `outcomes.separate_package` prior to this pull request.